src/components/Footer/index.tsx

```diff
@@ -165,33 +184,29 @@ const Footer = () => {
             <CustomAccordion items={accordionItems} />
           </div>
           <div className={styles.Box} data-testid='get-connected'>
-            <Text type='subtitle-1' as='h3' className={styles.SectionTitle}>
+            <Heading.H5 className={styles.SectionTitle2}>
               <Translate>Get connected</Translate>
-            </Text>
+            </Heading.H5>
             <p className={styles.SectionContent}>
-              <Translate>Discuss ideas and share solutions with developers worldwide.</Translate>
+              <Text size='sm'>
+                <Translate>Discuss ideas and share solutions with developers worldwide.</Translate>
+              </Text>
             </p>
             <div className={styles.CommunityButton} data-testid='community-button'>
               <Button
-                color='secondary'
+                variant='secondary'
+                color='black'
```

Comment:
you can pass the `fullWidth` prop to `join our community` button to have the alignment issue fixed
```suggestion
                variant='secondary'
                color='black'
                fullWidth
```